fastlane/Fastfile

```diff
@@ -140,33 +146,33 @@ end
   # Example:
   # bundle exec fastlane update_appstore_strings version:1.1
   #####################################################################################
-  desc "Updates the AppStoreStrings.pot file with the latest data"
-  lane :update_appstore_strings do | options |
-    prj_folder = Pathname.new(File.join(Dir.pwd, "..")).expand_path.to_s
-    source_metadata_folder = File.join(prj_folder, "fastlane/appstoreres/metadata/source")
+  desc 'Updates the AppStoreStrings.pot file with the latest data'
+  lane :update_appstore_strings do |options|
+    prj_folder = Pathname.new(File.join(Dir.pwd, '..')).expand_path.to_s
+    source_metadata_folder = File.join(prj_folder, 'fastlane/appstoreres/metadata/source')
```

Comment:
We should pass each path component as a separate param to `File.join` (after all that's the whole point of `File.join` to construct the path for us appropriately)

```suggestion
    source_metadata_folder = File.join(prj_folder, 'fastlane', 'appstoreres', 'metadata', 'source')
```